lib/portlayer/event/collector/vsphere/collector.go

```diff
@@ -159,7 +163,21 @@ func evented(ec *EventCollector, page []types.BaseEvent) {
 		return
 	}
 
+	if len(page) == 0 {
+		return
+	}
+
+	// skip events already seen
```

Comment:
Not sure if we can make use of it in this case, but CreateCollectorForEvents takes a filter param:
http://pubs.vmware.com/vsphere-60/topic/com.vmware.wssdk.apiref.doc/vim.event.EventFilterSpec.html

We already use it filter by entity.  It also has a time field: http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.wssdk.apiref.doc/vim.event.EventFilterSpec.ByTime.html

Guessing we could set the beginTime to that of the last event timestamp we had processed, and avoid fetching these "stale" events at all.  Would require a govmomi change though: https://github.com/vmware/govmomi/blob/master/event/processor.go#L53

cc @cgtexmex 

But functionally, this should be fine as-is.